trino/etc/secrets/certificate_with_key.pem

```diff
@@ -0,0 +1,57 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Would we ever need to regenerate those? How long are these valid?

Reply:
It is valid for 10 years, let me change to 100.